app/UI/conversation-list.ts

```diff
@@ -18,6 +19,7 @@ export class DM_List implements ConversationListRetriever, NewMessageChecker, Us
 
     constructor(
         public readonly ctx: NostrAccountContext,
+        private readonly relayRecordGetter: RelayRecordGetter,
```

Comment:
This is bad because it introduced unnecessary coupling and dependency

Reply:
I try to improve it.